engine/dlib/src/dlib/message.cpp

```diff
@@ -1,10 +1,10 @@
 // Copyright 2020 The Defold Foundation
 // Licensed under the Defold License version 1.0 (the "License"); you may not use
 // this file except in compliance with the License.
-// 
+//
```

Comment:
I can't help laughing a little bit every time I see these changes :)

Reply:
Yeah, I keep thinking that we should update with the new license in one big swoop, but alas...